.github/workflows/build.yml

```diff
@@ -193,6 +193,8 @@ jobs:
 
       - name: Build nomad-builder image
         uses: docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4 # v6
+        env:
+          DOCKER_BUILD_SUMMARY: false
```

Comment:
Maybe you also need: `DOCKER_BUILD_RECORD_UPLOAD: false`? https://github.com/docker/build-push-action?tab=readme-ov-file#environment-variables

Reply:
I can try but the result of the build action seems clean? https://github.com/hashicorp/nomad/actions/runs/14472045048

Reply:
Good enough for me! :D